LIMS_IMAGE/web/templates/laboratory/home_page.html

```diff
@@ -1,19 +1,35 @@
 {% extends 'base.html' %}
+{% load static %} 
 <!-- Not having this here makes the interpreter mad -->
 {% block title %}Laboratory Home{% endblock %}
 <!-- Not having this here makes the interpreter mad -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    {% block additionalStyles %}
+    <link rel="stylesheet" href="{% static 'css/home.css' %}"/> 
+    {% endblock %}
+  </head>
 <body>
   {% block content%}
   <div class="container">
     <h1>Employee Home</h1>
-    <p>Welcome to the LIMS home page.</p>
     {% if user.is_authenticated %} Hi {{ user.first_name }}!
     <p><a href="{% url 'logout' %}">Log Out</a></p>
```

Comment:
since we have a nice log out button in the corner, we can remove this now
``` <p><a href="{% url 'logout' %}">Log Out</a></p>```

Reply:
I will remove the Hi part too since we have that in the corner with the log out button.